contracts/contracts/utils/InitializableAbstractStrategy.sol

```diff
@@ -124,6 +128,10 @@ contract InitializableAbstractStrategy is Initializable, Governable {
         external
         onlyGovernor
     {
+        emit RewardLiquidationThresholdUpdated(
```

Comment:
If it is intended to be able to reconstruct all values that these are set to from the events, should we fire these events in the initializer as well?